src/js/containers/App/App.js

```diff
@@ -4,6 +4,7 @@ import { FormattedMessage, injectIntl, IntlProvider } from 'react-intl';
 import { GatewayDest, GatewayProvider } from 'react-gateway';
 
 import { LayoutContainer, LayoutRow, LayoutColumn } from 'cf-component-layout';
+import { Flex } from 'cf-component-flex';
```

Comment:
Lets talk about this on a call on Monday.  We shouldn't be mixing the use of `cf-component-layout` and `cf-component-flex`.  They are different solutions to the same problem - structuring tabular data.  We should be using one or the other.